content/blog/elsewhere/sass-color-draft.md

```diff
@@ -0,0 +1,87 @@
+---
+title: Sass Color Spaces (Request for Comments)
+sub: A new proposal for color-management in Sass
+venue: Sass Blog
+url: https://sass-lang.com/blog/request-for-comments-color-spaces
+date: 2022-09-22
+author: miriam
+image:
+  src: blog/2022/sass-color.jpg
+  alt: |
+    Sass logo in black
+    on top of bright oklch color gradient
+tags:
+  - CSS
+  - Sass
+action:
+  text: Read the summary & proposal »
+  url: https://sass-lang.com/blog/request-for-comments-color-spaces
+summary: |
+  There’s been a lot of exciting work
+  in the CSS color specifications lately,
+  and the new features are starting to land in browsers --
+  so we’ve been preparing to add support in Sass as well.
+  My proposal for that
+  is published and ready for public feedback!
+---
+
+I'm very excited
+about the new color features
+starting to roll out in CSS --
+and similarly excited to make them available
+(with some extra tooling)
+in Sass as well.
+This will allow us to:
+
+- Define colors in new color spaces like `oklch` & `display-p3`
```

Comment:
would links to documentation about `oklch` and `display-p3` be helpful for readers who are unfamiliar with those spaces?

Reply:
Maybe? But I didn't know of any useful direct links, and figured the link to Bramus' article below might be good enough? Do you have better ideas for links here?

Reply:
> Maybe? But I didn't know of any useful direct links, and figured the link to Bramus' article below might be good enough? Do you have better ideas for links here?

I don't! Yeah, I'm not really seeing good options for direct links either. This is ok with no links 👍🏽